### ISSUE TITLE
Patch Not working : Eiyuden Chronicle Hundred Heroes [21.9 Ultrawide v1.0.3]

### ISSUE THREAD

**Issue by Boyyatio** [Fl4sh9174/Switch-Ultrawide-Mods#11]
Dear Team developer lover, I try two of this patch one Android and PC also, but still not working ultrawide, Thanks. 

Eiyuden Chronicle Hundred Heroes [21.9 Ultrawide v1.0.3]

**Comment by Fl4sh9174** [comment]
Because i forgot to uploaded the aspect ratio mods, now i just uploaded all the mods, test and let me know if its working.

**Comment by Boyyatio** [comment]
After i tested on your two of new file just uploaded, look like still not working.

Can i request you to make ultrawide game as below game ?
1. FIFA 23 Legacy Edittion
2. Dragon's Dogma: Dark Arisen
3. Tales of Kenzera ZAU

![Screenshot_suyu Debug Release_20240425_113020.jpg](https://github.com/Fl4sh9174/Switch-Ultrawide-Mods/assets/165496235/67afef87-c325-4fa5-a481-d1b254f52266)

**Comment by Fl4sh9174** [comment]
Check the update version you are using, tested it here and all the mods are working fine...

**Comment by Boyyatio** [comment]
Okay dear ! Let me try again. I don't know what happened too, because other ultrawide mod games are working well.
